sdk/avalon_sdk/worker/worker_details.py

```diff
@@ -41,6 +41,19 @@ class WorkerType(Enum):
     ZK = 3
 
 
+@unique
+class WorkerMode(Enum):
```

Comment:
Do we need this enum in worker_details. Why should a client which uses worker_details know about worker mode ?

Reply:
This is not needed in case of client, but it will used in enclave manager. We have many worker related enums in this file which are already being used in enclave manager.

Reply:
Then these should be moved from SDK to enclave manager

Reply:
But some worker related enums are also used by clients.